geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/common/commands/HelpCommand.scala

```diff
@@ -6,13 +6,13 @@
 * http://www.opensource.org/licenses/apache2.0.php.
 *************************************************************************/
 
-package org.locationtech.geomesa.tools.commands
+package org.locationtech.geomesa.tools.common.commands
 
 import java.util
 
 import com.beust.jcommander.{JCommander, Parameter, Parameters}
-import org.locationtech.geomesa.tools.Runner.commandUsage
-import org.locationtech.geomesa.tools.commands.HelpCommand.HelpParameters
+import org.locationtech.geomesa.tools.accumulo.AccumuloRunner.commandUsage
+import org.locationtech.geomesa.tools.common.commands.HelpCommand.HelpParameters
```

Comment:
FYI just ran this:
```
ahulbert@doubletrouble:/tmp/geomesa-tools-1.2.2-SNAPSHOT$ bin/geomesa-kafka help
Warning: GEOMESA_HOME is not set, using /tmp/geomesa-tools-1.2.2-SNAPSHOT
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/accumulo/core/client/TableNotFoundException
        at org.locationtech.geomesa.tools.accumulo.commands.TableConfCommand.<init>(TableConfCommand.scala:33)
        at org.locationtech.geomesa.tools.accumulo.AccumuloRunner$.<init>(AccumuloRunner.scala:30)
        at org.locationtech.geomesa.tools.accumulo.AccumuloRunner$.<clinit>(AccumuloRunner.scala)
        at org.locationtech.geomesa.tools.common.commands.HelpCommand.execute(HelpCommand.scala:27)
        at org.locationtech.geomesa.tools.common.Runner$class.main(Runner.scala:26)
        at org.locationtech.geomesa.tools.kafka.KafkaRunner$.main(KafkaRunner.scala:15)
        at org.locationtech.geomesa.tools.kafka.KafkaRunner.main(KafkaRunner.scala)
Caused by: java.lang.ClassNotFoundException: org.apache.accumulo.core.client.TableNotFoundException
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
```

can you check the imports, etc. and make sure we don't need accumulo installed to run the kafka tool?